### PR TITLE
fuppes: add recipe

### PR DESCRIPTION
**Pull request by Ra-V2**
It seems we don't have recipe for libev what is needed to build this (or I'm wrong?).
For now this is WIP.

### CODE REVIEW

File: media-video/fuppes/fuppes-1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,56 @@
+SUMMARY="Free UPnP A/V media server"
+DESCRIPTION="FUPPES is a free, multiplatform UPnP A/V Media Server. \
+FUPPES supports a wide range of UPnP MediaRenderers as well as on-the-fly \
+transcoding of various audio, video and image formats. FUPPES also includes \
+basic DLNA support."
+HOMEPAGE="https://github.com/uvoelkel/fuppes/"
+COPYRIGHT="1989-1991 Free Software Foundation"
```
Comment (scottmc) [inline media-video/fuppes/fuppes-1.0.recipe]: 2007-2009 Ulrich Völkel
remove the FSF one as that is likely just from one of the build tools
I normal run grep on the source directory searching for "copyright" and then investigate the files that come up in that list.
`
grep -nri "copyright" 
`

**Comment by Begasus** [comment]
Needs SECONDARY_ARCHITECTURES, libevent isn't available for gcc2

**Comment by Begasus** [comment]
> It seems we don't have recipe for libev what is needed to build this (or I'm wrong?).
> For now this is WIP.

Indeed missing ... http://software.schmorp.de/pkg/libev.html

**Comment by Ra-V2** [comment]
@Begasus https://github.com/haikuports/haikuports/pull/4641

**Comment by Begasus** [comment]
> @Begasus #4641

Left some comments there

File: media-video/fuppes/fuppes-1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext
+	cmd:git
+	cmd:libtoolize
```
Comment (Begasus) [inline media-video/fuppes/fuppes-1.0.recipe]: Missing:
```diff
- 	cmd:libtoolize
+ 	cmd:libtoolize$secondaryArchSuffix
```

**Comment by Begasus** [comment]
configure checks for other libraries we already have, maybe you can add them in REQUIRES and BUILD_REQUIRES?

**Comment by Begasus** [comment]
Needs GLOBAL_WRITABLE_FILES for the cfg files

File: media-video/fuppes/fuppes-1.0.recipe [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	lib:libmpcdec$secondaryArchSuffix
+	lib:libmp4v2$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libtag$secondaryArchSuffix
+	lib:libuuid$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
```
Comment (Begasus) [inline media-video/fuppes/fuppes-1.0.recipe]: libs disabled by default (can be left out):
libmad, libfaad

File: media-video/fuppes/fuppes-1.0.recipe [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+	lib:libavutil$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libev$secondaryArchSuffix
+	lib:libevent$secondaryArchSuffix
+	lib:libexiv2$secondaryArchSuffix
+	lib:libfaad$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libmad$secondaryArchSuffix
+	lib:libMagickWand_7.Q16HDRI$secondaryArchSuffix
```
Comment (Begasus) [inline media-video/fuppes/fuppes-1.0.recipe]: Maybe lib:libMagickWand_6.Q16 (from what I see lib:libMagickWand_7.Q16HDI doesn't provide the headers?)

File: media-video/fuppes/fuppes-1.0.recipe [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext
+	cmd:git
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
```
Comment (Begasus) [inline media-video/fuppes/fuppes-1.0.recipe]: Not required but saves some output on the screen :)
```diff
+	export CPPFLAGS="-Wno-deprecated"
```

**Comment by Begasus** [comment]
With the latest changes it requires a _devel package

**Comment by korli** [comment]
This software looks unmaintained since 2014. It's probably not worth the effort: https://repology.org/project/fuppes/versions

**Comment by Begasus** [comment]
@korli close this one?

**Pull request closed without merging**